OsStub/BaseCryptLibMbedTls/Hash/CryptSha2.c

```diff
@@ -0,0 +1,403 @@
+/** @file
+  SHA-2 Digest Wrapper Implementations over OpenSSL.
+
+  This file includes SHA256, SHA384, SHA512, all part of SHA-2 family.
+
+Copyright (c) 2009 - 2016, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include "InternalCryptLib.h"
+#include <mbedtls/sha256.h>
+#include <mbedtls/sha512.h>
+
+enum InternalSha2Type{_SHA2_256, _SHA2_384, _SHA2_512};

```

Comment:
This coding style is not acceptable.

Please use:
typedef enum {
  _SHA2_256,
  _SHA2_384,
  _SHA2_512,
} INTERNAL_SHA2_TYPE;